documentation/modules.md

```diff
@@ -142,7 +142,85 @@ modulepath = site:dist:modules:$basemodulepath:.modules
 
 You can learn more about the module changes introduced in Bolt 2.23.0 in the
 [September 2020 developer update](./developer_updates.md#september-2020).
+
+## Migrate a project to use module management
+
+Existing Bolt projects can be easily updated to use the module management
+feature. In most cases, Bolt can update your project to use this feature
+for you. However, project's with more complex module requirements may
+require manual updating.
+
+If your project requires any of the following, you should update your
+project manually:
+
+- Modules installed through a proxy
+- Modules installed from a Forge other than the [Puppet
+  Forge](https://forge.puppet.com)
+- Modules installed from a git repository other than a public GitHub repository
  
+### Automated migration
+
+For information on updating your project using Bolt, see [Migrate a Bolt
+project](projects.md#migrate-a-bolt-project).
+
+### Manual migration
```

Comment:
Let's move this section to the installing modules page under the "Manually manage a project's modules" section. I think we can treat it as a prerequisite to manually managing modules.